app/usecase/user_interactor.go

```diff
@@ -1,10 +1,14 @@
 package usecase
 
 import (
+	"errors"
```

Comment:
Same as user_repository. go. You shouldn't error handle in the interactor. error handle should be in controller.
But I understand what you want to do. So right now just leave TODO: is better.

Reply:
got rid of it!